polaris-core/src/test/java/org/apache/polaris/service/storage/azure/AzureCredentialStorageIntegrationTest.java

```diff
@@ -268,7 +268,7 @@ public void testGetSubscopedTokenWrite(
                 service, allowedPrefix));
     Map<PolarisCredentialProperty, String> credsMap =
         subscopedCredsForOperations(
-            /* allowedReadLoc= */ realmId,
+            /* allowedReadLoc= */ realm,
```

Comment:
This comment looks incorrect?

Reply:
Fixed. The mismatch was introduced by #741.